app/src/interfaces/_system/system-input-translated-string/input-translated-string.vue

```diff
@@ -124,6 +125,12 @@ function openNewCustomTranslationDrawer() {
 	menuEl.value.deactivate();
 	isCustomTranslationDrawerOpen.value = true;
 }
+
+const newTranslationDefaults = computed(() => ({
+	language: getCurrentLanguage(),
+	value: localValue.value,
+	key: snakeCase(localValue.value ?? ''),
+}));
```

Comment:
I think we should only prefill the key and value fields if it is a plain value (not already translated), as otherwise you'd end up with such values:
<img width="200" src="https://github.com/user-attachments/assets/1746f66d-3cb3-40a2-b4d9-22be7bc8fc90" />

```suggestion
const newTranslationDefaults = computed(() => {
	const defaults = {
		language: getCurrentLanguage(),
	};

	if (localValue.value && !localValue.value.startsWith(translationPrefix)) {
		Object.assign(defaults, {
			key: snakeCase(localValue.value),
			value: localValue.value,
		});
	}

	return defaults;
});
```

Reply:
Thanks <3 Good catch!